cats/cats-core/cats-core.gradle

```diff
@@ -3,7 +3,7 @@ dependencies {
 
   implementation "org.slf4j:slf4j-api"
   implementation "com.fasterxml.jackson.core:jackson-annotations"
-  implementation "org.codehaus.groovy:groovy-all"
+  implementation "org.codehaus.groovy:groovy"
```

Comment:
IF we change this to API, it should expose it downstream (aka cats-core being a java librayr, no need for implementation downstream)